### PR TITLE
Better error message and sanity check for signed URL generation.

### PR DESCRIPTION
**Pull request by DailyDreaming**

### CODE REVIEW

**Comment by DailyDreaming** [comment]
Tests are failing because the firecloud library is broken on python3.10: https://github.com/DataBiosphere/terra-notebook-utils/issues/390

File: terra_notebook_utils/drs.py [hunk 1]
```diff
@@ -121,6 +121,16 @@ def access(drs_url: str,
                                 key=info.key,
                                 sa_credentials=info.credentials,
                                 requester_pays_user_project=billing_project)
+        # one last sanity check
+        response = requests.get(url, headers={'Range': 'bytes=0-1'})
+
+    if response.status_code >= 400:
+        raise DRSResolutionError(f'Could not generate a valid signed URL for {drs_url} !  Make sure you have access '
```
Comment (mbaumann-broad) [inline terra_notebook_utils/drs.py:128]: @DailyDreaming - nice job with this error message, thanks!
I like it except for the exclamation point. Please remove that. :)

**Review by mbaumann-broad — CHANGES_REQUESTED** [review]

File: terra_notebook_utils/drs.py (hunk 1 above)
Comment (DailyDreaming) [reply terra_notebook_utils/drs.py:128]: Ha, alright, thanks for the review.

**Comment by DailyDreaming** [comment]
@mbaumann-broad Changes made.

**Review by mbaumann-broad — APPROVED** [review]
LGTM, thank you @DailyDreaming!

**Comment by DailyDreaming** [comment]
@mbaumann-broad Thanks for the reviews!

**Pull request merged**